multigeiger/multigeiger.ino

```diff
@@ -66,7 +66,7 @@ void setup() {
   setup_webconf();
   setup_transmission(VERSION_STR, ssid);
 
-  if (PLAY_SOUND)
+  if (parameterTrue(startSound))
```

Comment:
calling parameterTrue() is slightly ugly and always causing overhead. use a bool variable.

Reply:
I think that results in an unnecessary duplication. You have to copy the bool variables to the config variables and back again.

Reply:
usually when dealing with user input one needs some validation step (and rejection if invalid, letting the user try again).
and as the validation has to deal with types and valid values/ranges anyway, it usually can transform the string to the desired type also.

Reply:
most codebases i have seen yet do a one-time read, validate, convert step when reading their text-based config (usually a text file in some format) and after that only use the appropriate types (bool, int, string, ...) but not the config text any more.

in our case here, the iotWebConf variables have to be considered our "text config" and the additional set of appropriately typed variables would contain the stuff we use from the loop.

if we immediately validate at the form input step and reject any form that has invalid data, we can immediately get the user to edit and resubmit the form, until we get input that validates.

Reply:
OK, sounds good. Would you please try that?